tools/driver/p4c.py

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env python
+
+from p4c.main import main
+
+if __name__ == '__main__':
+    main()
```

Comment:
The `p4c` driver program should not have a .py extension (so it can be invoked directly as `p4c ...args`, rather than `p4c.py ...args`)